task/yq/0.1/yq.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: yq
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.24.1"
+    tekton.dev/tags: yq
+    tekton.dev/displayName: "YQ replace"
+spec:
+  description: >-
+    This task can be used to replace fields in YAML files. For example for altering helm charts on GitOps repos.
+  workspaces:
+    - name: shared-workspace
+      description: A workspace that contains the fetched git repository.
+  params:
+    - name: file
+      type: string
+    - name: expression
+      type: string
+  steps:
+    - name: replace-in-yaml
+      image: docker.io/mikefarah/yq:4@sha256:34f1d11ad51dc4639fc6d8dd5ade019fe57cf6084bb6a99a2f11ea522906033b #tag: 4, amd
+      workingDir: $(workspaces.shared-workspace.path)
+      command:
+        - /usr/bin/yq
+      args:
+        - eval
+        - -i
+        - $(params.expression)
+        - $(workspaces.shared-workspace.path)/$(params.file)
```

Comment:
I guess we can avoid `$(workspaces.shared-workspace.path)` over here as the `workingDir` is already set

Reply:
Done.